numpy/lib/tests/test_histograms.py

```diff
@@ -366,10 +364,9 @@ def do_precision_upper_bound(self, float_small, float_large):
 
         # previously crashed
         count, x_loc = np.histogram(arr, bins=1, range=range)
-        assert_equal(count, [1])
+        assert_equal(count, [0])
 
-        # gh-10322 means that the type comes from arr - this may change
-        assert_equal(x_loc.dtype, float_small)
+        assert_equal(x_loc.dtype, float_large)
```

Comment:
@eric-wieser these histogram changes look right to you?  What was previously rounded to the small bin, now ends up with the large dtype because it is explicit, I guess.

Reply:
FWIW, I think this is logical as long as one doesn't special-case histogram in treating its arguments asymmetrically.